### PR TITLE
doc: merge announcements and blogs into news

### LINKED ISSUE #2987: docs: refactor the website
I think we should reconstruct the document site as follows:
- [ ] Merge `announcements` and `blogs`
- [x] Move "Get Involved" to menu, and it should not be versioned.

cc @Xunzhuo @arkodg

### PR DESCRIPTION
**Pull request by shawnh2**
**What type of PR is this?**

**What this PR does / why we need it**:

before:

image

after:

image

**Which issue(s) this PR fixes**:

Fixes #2987

### CODE REVIEW

**Comment by shawnh2** [comment]
HI, @wilsonwu can you also do the same refactor for the chinese doc? TIA

**Comment by wilsonwu** [comment]
> HI, @wilsonwu can you also do the same refactor for the chinese doc? TIA

No problem, I will sync ASAP after it is merged.

File: site/content/zh/contributions/DOCS.md [hunk 1]
```diff
@@ -5,8 +5,6 @@ description: "本节讲述 Envoy Gateway 文档的开发工作。"
 
 {{% alert title="注意" color="warning" %}}
 我们已将 Envoy Gateway 文档从 **Sphinx** 迁移到 **Hugo**。
-
-阅读博客：[Welcome to new website!](/blog/2023/10/08/welcome-to-new-website/)
```
Comment (shawnh2) [inline site/content/zh/contributions/DOCS.md:9]: remove here just to pass doc-lint in ci, shall undo it when translating chinese doc

**Comment by wilsonwu** [comment]
Not sure our site support alias link or not? For example if we change a blog link from `/blog/2023/10/08/welcome-to-new-website/` to `/news/blogs/new-website/new-website`, we can add below info for the blog make the old link still work:
```
aliases:
  - blog/2023/10/08/welcome-to-new-website/
```

File: site/content/zh/contributions/DOCS.md (hunk 1 above)
Comment (wilsonwu) [reply site/content/zh/contributions/DOCS.md:9]: I will remember it and add it back with the sync to zh PR.

**Comment by shawnh2** [comment]
> Not sure our site support alias link or not? For example if we change a blog link from `/blog/2023/10/08/welcome-to-new-website/` to `/news/blogs/new-website/new-website`, we can add below info for the blog make the old link still work:
> 
> ```
> aliases:
>   - /blog/2023/10/08/welcome-to-new-website/
> ```

Great input, let me try this!

**Review by arkodg — APPROVED** [review]
thanks for improving the presentation !

**Review by zirain — APPROVED** [review]

**Pull request merged**